AUTHORS

```diff
@@ -0,0 +1,2 @@
+- Kitamura Satoshi <with.no.parachute@gmail.com>
```

Comment:
Yeah let's add authors. This is a requirements from our legal team to add Google.